pkgs/applications/misc/mockbob/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  name = "mockbob-${version}";
+  version = "2.0.0";
+
+  src = fetchFromGitHub {
+    owner = "j-brn";
+    repo = "mockbob";
+    rev = "${version}";
+    sha256 = "07ygpdak4h630wvk4xjsqwphvcbrif39zkmxdagj5ic7rkahii26";
+  };
+
+  cargoSha256 = "0n94qpasvplcj1vjgqgvxg5jpczij71hfxk4mvxy9q3wa522q7mq";
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/j-brn/mockbob/tree/master";
+    description = "cOmMaNdLiNe uTiLiTy tO MoCk sEnTeNcEs lIkE ThIs.";
```

Comment:
I know that this is the purpose of this pacakge, but please keep the description readable and nice.